compiler/test/formatter_outputs/application.gr

```diff
@@ -81,5 +81,5 @@ let preExistingObjectsWithRefCountMismatch2 = Map.make(): Map.Map<
     Number,
     Number,
     Number
-  )
+  ),
```

Comment:
Yeah, this is where I don't want a trailing comma.

Reply:
Glad to hear that, I wondered why you wanted it, it looks ugly